<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1359</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revise Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="5" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Files list</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="lFilesList">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbSelectAll">
          <property name="text">
           <string>Select all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bApply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" rowspan="2" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Signals</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Processing parameters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Resample frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbResampleFrequency">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Detrend frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbDentendEdge"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Start time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dtStartTime">
             <property name="displayFormat">
              <string>dd.MM.yyyy HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Stop time</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="dtStopTime">
             <property name="displayFormat">
              <string>dd.MM.yyyy HH:mm:ss</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bUpdateSignalData">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2">
         <widget class="PlotWidget" name="signalsPlot"/>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Component</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbSignalComponents"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Time marker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="sbSignalTimeMarker">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>99990000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Render factor, %</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbRenderFactor">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Average spectrums</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="GraphicsLayoutWidget" name="spectrumsPlot"/>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Visualization parameters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Minimal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbMinVisualFrequency">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Maximal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbMaxVisualFrequency">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbOriginalSpectrums">
             <property name="text">
              <string>O&amp;riginal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSmoothSpectrums">
             <property name="text">
              <string>Smoo&amp;th</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Processing parameters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Window size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbWindowSizes">
             <property name="currentIndex">
              <number>3</number>
             </property>
             <item>
              <property name="text">
               <string>1024</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2048</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4096</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8192</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16384</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>32768</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>65536</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Overlap size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbOverlapSize">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="value">
              <number>4096</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Marmett filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbMarmettFilter">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="singleStep">
              <number>2</number>
             </property>
             <property name="value">
              <number>7</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Median filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbMedianFilter">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="singleStep">
              <number>2</number>
             </property>
             <property name="value">
              <number>7</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bLoadSpectrums">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Component</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbSpectrumComponents"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Correlations</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Processing parameters</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Minimal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbMinCorrelationFrequency">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Maximal frequency</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="sbMaxCorrelationFrequency">
             <property name="maximum">
              <double>1000.000000000000000</double>
             </property>
             <property name="value">
              <double>10.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbUsingSmooth">
             <property name="text">
              <string>using smooth spectrums</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bLoadCorrelations">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Component</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbCorrelationComponents"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="GraphicsLayoutWidget" name="correlationsPlot"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1359</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>O&amp;ptions</string>
    </property>
    <addaction name="aExport"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>sbResampleFrequency</tabstop>
  <tabstop>sbDentendEdge</tabstop>
  <tabstop>dtStartTime</tabstop>
  <tabstop>dtStopTime</tabstop>
  <tabstop>bUpdateSignalData</tabstop>
  <tabstop>cbSignalComponents</tabstop>
  <tabstop>sbSignalTimeMarker</tabstop>
  <tabstop>lFilesList</tabstop>
  <tabstop>cbSelectAll</tabstop>
  <tabstop>bApply</tabstop>
  <tabstop>signalsPlot</tabstop>
  <tabstop>sbOverlapSize</tabstop>
  <tabstop>sbMarmettFilter</tabstop>
  <tabstop>sbMedianFilter</tabstop>
  <tabstop>bLoadSpectrums</tabstop>
  <tabstop>sbMinVisualFrequency</tabstop>
  <tabstop>sbMaxVisualFrequency</tabstop>
  <tabstop>rbOriginalSpectrums</tabstop>
  <tabstop>rbSmoothSpectrums</tabstop>
  <tabstop>cbSpectrumComponents</tabstop>
  <tabstop>spectrumsPlot</tabstop>
  <tabstop>sbMinCorrelationFrequency</tabstop>
  <tabstop>sbMaxCorrelationFrequency</tabstop>
  <tabstop>cbUsingSmooth</tabstop>
  <tabstop>bLoadCorrelations</tabstop>
  <tabstop>cbCorrelationComponents</tabstop>
  <tabstop>correlationsPlot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
